src/local/butler/scripts/migration/jobs_keywords.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Build keyword attributes for every job."""
+
+from datastore import data_types
+from google.cloud import ndb
+
+
+def execute(args):
+  """Build keywords for jobs."""
+  jobs = list(data_types.Job.query())
+  if args.non_dry_run:
+    try:
+      ndb.put_multi(jobs)
+      print("Done building keywords for jobs.")
+    except Exception:
+      print("Error during building keywords for jobs.")
```

Comment:
Better to just let the exception surface normally, rather than handling. As is, the error message is not very helpful as it hides the stacktrace.

Reply:
done